### ISSUE TITLE
Receiver tuned to 151.5 MHz reporting tags from 166.380 MHz

### ISSUE THREAD

**Issue by leberrigan** [MotusDev/Motus-TO-DO#520]
User from project 227 has reported this behaviour for their receiver which has been _correctly_ tuned to 151.5 MHz (deployment.txt was modified). Only two tags were detected on this receiver, but both are tuned to 166.380 MHz: https://motus.org/data/receiverDeploymentDetections?id=5359

John connected the receiver online so he could help troubleshoot an issue for the user which may have resulted in these data being produced before the receiver was registered. https://sgdata.motus.org/status?jobID=726505

Jeremy, could you reprocess these data to see if that clears things up?

**Comment by Hussell** [comment]
Ok, I've set a default frequency for this project and rerun both their receivers. Detections of North American tags have disappeared.

**Comment by Hussell** [comment]
I did a search for projects with deployments with longitude > -30 deg. and found 3 more projects outside the Americas without a default frequency set. I've set those, and rerun the receivers affected. There may still be existing projects with no deployment metadata, but hopefully not. I'll rerun this query from time-to-time in the future, but I can't guarantee this won't happen again.

Only one of the three other projects had uploaded any receiver data, and only one receiver had new detections after being rerun: SG-5305RPI3BB5A in project 223.
